include/eye.hpp

```diff
@@ -85,191 +80,162 @@ class Eye
-    sprite_eye.fillScreen(TFT_WHITE);
-    sprite_eyeball.pushSprite(&sprite_eye, 0, 0, TFT_WHITE);
-
-    sprite_iris.pushSprite(&sprite_eye, 0, 5, TFT_WHITE);
-    sprite_pupil.pushSprite(&sprite_eye, (int)(scale * dx), (int)(scale * dy), TFT_WHITE);
-    sprite_reflex.pushSprite(&sprite_eye, (int)(scale * dx) + rx, (int)(scale * dy) + ry, TFT_WHITE);
-    sprite_upperlid.pushSprite(&sprite_eye, 0, upperlid_y_arr[surprised_level], TFT_WHITE);
-    
-    sprite_eye.pushRotateZoom(&lcd, lcd.width() >> 1, lcd.height() >> 1, 0, zoom_ratio, zoom_ratio, TFT_WHITE);
-  }
-
-  // 眠い目の描画
-  void sleepy(float dx = 0.0, float dy = 0.0, int sleepy_level = 0, float scale = 10.0, float random_scale = 5.0)
-  {
-    int upperlid_y_arr[] = {- 70, - 70, -60, 0, 0, 0, -60, -70, -70, - 70};
-    long rx = (int)(random_scale * random(100) / 100);
-    long ry = (int)(random_scale * random(100) / 100);
-    
-    sprite_eye.clear();
-    sprite_eye.fillScreen(TFT_WHITE);
-    sprite_eyeball.pushSprite(&sprite_eye, 0, 0, TFT_WHITE);
-    sprite_iris.pushSprite(&sprite_eye, -10, 15, TFT_WHITE);
-    
-    sprite_pupil.pushSprite(&sprite_eye, (int)(scale * dx) - 10, (int)(scale * dy) + 15, TFT_WHITE);
-    sprite_upperlid.pushSprite(&sprite_eye, 0, upperlid_y_arr[sleepy_level]);
-
-    sprite_eye.pushRotateZoom(&lcd, lcd.width() >> 1, lcd.height() >> 1, 0, zoom_ratio, zoom_ratio, TFT_WHITE);
-  }
-
-  // 怒った目の描画
-  void angry(float dx = 0.0, float dy = 0.0, int angry_level = 0, float scale = 10.0, float random_scale = 5.0)
-  {
-    long rx = (int)(random_scale * random(100) / 100);
-    long ry = (int)(random_scale * random(100) / 100);
-    
-    sprite_eye.clear();
-    sprite_eye.fillScreen(TFT_WHITE);
-    sprite_eyeball.pushSprite(&sprite_eye, 0, 0, TFT_WHITE);
-    sprite_iris.pushSprite(&sprite_eye, 0, 0, TFT_WHITE);
-    
-    sprite_pupil.pushSprite(&sprite_eye, (int)(scale * dx), (int)(scale * dy), TFT_WHITE); // 瞳孔をランダムに動かす
-    sprite_reflex.pushSprite(&sprite_eye, (int)(scale * dx) + rx, (int)(scale * dy) + ry + 10, TFT_WHITE); // 光の反射をランダムに動かす
-    
-    sprite_upperlid.pushSprite(&sprite_eye, 0, -10, TFT_WHITE);
-
+  void draw_updated_image(){
     sprite_eye.pushRotateZoom(&lcd, lcd.width() >> 1, lcd.height() >> 1, 0, zoom_ratio, zoom_ratio, TFT_WHITE);
   }
+};
 
-  // 悲しい目の描画
-  void sad(float dx = 0.0, float dy = 0.0, int sad_level = 0, float scale = 10.0, float random_scale = 5.0)
-  {
-    long rx = (int)(random_scale * random(100) / 100);
-    long ry = (int)(random_scale * random(100) / 100);
-    
-    sprite_eye.clear();
-    sprite_eye.fillScreen(TFT_WHITE);
-    sprite_eyeball.pushSprite(&sprite_eye, 0, 0, TFT_WHITE);
-    sprite_iris.pushSprite(&sprite_eye, 0, 0, TFT_WHITE);
-    
-    sprite_pupil.pushSprite(&sprite_eye, (int)(scale * dx), (int)(scale * dy), TFT_WHITE); // 瞳孔をランダムに動かす
-    sprite_reflex.pushSprite(&sprite_eye, (int)(scale * dx) + rx, (int)(scale * dy) + ry + 10, TFT_WHITE); // 光の反射をランダムに動かす
+class EmotionManager
```

Comment:
Please split this class definition into another header like `emotion_manager.hpp`